admin/resources/js/components/IntlContainer.tsx

```diff
@@ -1,11 +1,29 @@
 import React from "react";
 import { IntlProvider } from "react-intl";
 
+export function loadLocaleData(locale: string) {
```

Comment:
It looks like it has broken bundlewatch since it needs the French file.  Perhaps you need to make similar changes to the bundlewatch.yml file like you did here for the storybook.yml?
![image](https://user-images.githubusercontent.com/8978655/150018813-dbb7caa5-f9c6-4bd1-8557-d507b5f0f79a.png)

Reply:
Lol, well it completes now, at least. 👏